src/sklearn_evaluation/plot/__init__.py

```diff
@@ -2,32 +2,35 @@
 Plotting functions
 """
 
-from .classification import (
+from sklearn_evaluation.plot.classification import (
     confusion_matrix,
     feature_importances,
     precision_at_proportions,
     ConfusionMatrix,
 )
 
-from .roc import roc, ROC
-from .precision_recall import precision_recall
-from .grid_search import grid_search
-from .validation_curve import validation_curve
-from .learning_curve import learning_curve
-from .metrics import metrics_at_thresholds
-from .clustering import (
+from sklearn_evaluation.plot.roc import roc, ROC
+from sklearn_evaluation.plot.precision_recall import precision_recall
+from sklearn_evaluation.plot.grid_search import grid_search
+from sklearn_evaluation.plot.validation_curve import validation_curve
+from sklearn_evaluation.plot.learning_curve import learning_curve
+from sklearn_evaluation.plot.metrics import metrics_at_thresholds
+from sklearn_evaluation.plot.clustering import (
     elbow_curve,
     elbow_curve_from_results,
     silhouette_analysis,
     silhouette_analysis_from_results,
 )
-from .regression import residuals, prediction_error, cooks_distance
-from .target_analysis import target_analysis
-from .calibration import calibration_curve, scores_distribution
-from .classification_report import classification_report, ClassificationReport
-from .ks_statistics import ks_statistic
-from .cumulative_gain_lift_curve import cumulative_gain, lift_curve
-from .feature_ranking import Rank1D, Rank2D
+from sklearn_evaluation.plot.regression \
```

Comment:
@edublancas @idomic @WSShawn 
We changed the relative imports to absolute, but do we want to change these as well (the imports in `__init__.py`)?

it makes the code a bit longer and hard to maintain (e.g lint issue or if we will rename the `plot` directory).

in the [PEP8 guidelines ](https://peps.python.org/pep-0008/#imports) they write:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):

> ``` python
> import mypkg.sibling
> from mypkg import sibling
> from mypkg.sibling import example
> ```
> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:
> 
> ``` python
> from . import sibling
> from .sibling import example
> ```
> Standard library code should avoid complex package layouts and always use absolute imports.
>

Reply:
So we want to change the absolute back to explicit relative in _init_.py?

Reply:
Let's see what @edublancas and @idomic think

Reply:
let's keep them as absolute, we should be consistent (all absolute or all relative)